protobuf_definitions/message_formats.proto

```diff
@@ -763,11 +763,19 @@ message GuestPortInfo {
 }
 
 /**
- * Thickness measurement data from a Cygnus Thickness Gauge
+ * Thickness measurement data from a Cygnus Thickness Gauge.
  */
 message ThicknessGauge {
   float thickness_measurement = 1; // Thickness measurement of a steel plate
-  uint32 echo_count = 2; // Indicating the quality of the reading when invalid (0-3).
+  uint32 echo_count = 2; // Indicating the quality of the reading when invalid (0-3)
   uint32 sound_velocity = 3; // Speed of sound in the steel member
-  bool valid_measurement = 4; // Indicating if the measurement is valid
+  bool is_valid = 4; // Indicating if the measurement is valid
```

Comment:
I think that can work and it is good to have them alike. However, we just need to be aware that it could be interpreted as whether the sensor or connection is valid, not the measurement.

`is_measurement_valid` would be more self-explaining, especially since the message is called `ThicknessGauge` and not `ThicknessGaugeMeasurement`, including also data that is not specific for the measurement, i.e. `sound_velocity`

Reply:
Can rename to `is_measurement_valid` for this one 👍